buildrunner/validation/step.py

```diff
@@ -0,0 +1,106 @@
+"""
+Copyright 2023 Adobe
+All Rights Reserved.
+
+NOTICE: Adobe permits you to use, modify, and distribute this file in accordance
+with the terms of the Adobe license agreement accompanying it.
+"""
+
+from typing import Any, Dict, List, Optional, Union
+
+# pylint: disable=no-name-in-module
+from pydantic import BaseModel, Field
+
+
+class StepPypiPush(BaseModel):
```

Comment:
Should this forbid extra?

Reply:
I can do that. I just wasn't sure if the documentation was complete on that so I erred on the side of having it allow more. I'll add that in and hope for the best. :)